tests/ThreeWayMergeTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Relaxed\Merge\Test;
+
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
You should be able to remove this now when autoloading seems to be setup correctly.